forc/src/cli/commands/new.rs

```diff
@@ -43,6 +44,20 @@ pub(crate) fn exec(command: Command) -> Result<()> {
         path,
     } = command;
 
+    match &name {
```

Comment:
An alternative way of doing this is to handle it with clap's value_parser ([fuelup example](https://github.com/FuelLabs/fuelup/blob/07381e1597568d6f200d50c97e3780bab8edd4a3/src/commands/toolchain.rs#L29-L30)). However the changes look good as they are so this is just an alternative way of doing this.